play/src/front/Chat/Connection/Matrix/MatrixChatRoom.ts

```diff
@@ -117,18 +121,20 @@ export class MatrixChatRoom implements ChatRoom {
             })
         );
 
+        this.inMemoryEventsContent = new Map<EventId, MatrixEvent>();
+
         (async () => {
             if (matrixRoom.hasEncryptionStateEvent()) {
                 await matrixSecurity.initClientCryptoConfiguration();
             }
-        })().catch((error) => console.error(error));
+        })()
+            .catch((error) => console.error(error))
+            .then(async () => {
+                await this.initMatrixRoomMessagesAndReactions();
+            })
+            .catch((error) => console.error(error));
```

Comment:
Add Sentry logs here and just above.

Also, if we process this asynchronously, shouldn't we put this in a async "init" method called just after the constructor that we can actually await?